invokeai/frontend/web/src/features/metadata/util/parsers.ts

```diff
@@ -386,6 +386,7 @@ const parseIPAdapter: MetadataParseFunc<IPAdapterConfigMetadata> = async (metada
     clipVisionModel: 'ViT-H',
     controlImage: image?.image_name ?? null,
     weight: weight ?? initialIPAdapter.weight,
+    method: 'full',
```

Comment:
Shouldn't we recall this? Same for `clipVisionModel` (missed it in the prev PR)

Reply:
Probably should. I added it to the metadata. Wouldn't that retrieve it or is there something I missed?

Reply:
No it's hardcoded here to 'full', need to use the same pattern as the other IP metadata fields to retrieve it. Also suggest to fall back to `initialIPAdapter.method` instead of e.g. `'full'`

Reply:
Oh dang! My bad. I just added the default value there. Didn't realize it wasn't that. Will fix it up.